rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -57,8 +57,8 @@ extern "C" RocalTensor  ROCAL_API_CALL rocalResize(RocalContext context, RocalTe
                                                    unsigned resize_shorter = 0,
                                                    unsigned resize_longer = 0,
                                                    RocalResizeInterpolationType interpolation_type = ROCAL_LINEAR_INTERPOLATION,
-                                                   RocalTensorLayout rocal_tensor_output_layout = ROCAL_NHWC,
-                                                   RocalTensorOutputType rocal_tensor_output_type = ROCAL_UINT8);
+                                                   RocalTensorLayout output_layout = ROCAL_NONE,
+                                                   RocalTensorOutputType output_datatype = ROCAL_UINT8);
```

Comment:
Are these changes expected?

Reply:
This change is not a part of this PR, not sure why it shows up here. Need to investigate